content/en/llm_observability/setup/sdk/python.md

```diff
@@ -595,6 +608,31 @@ def llm_call():
     return completion
 {{< /code-block >}}
 
+### Exporting a span
+
+`LLMObs.export_span()` can be used to extract the span context from a span. You'll need to use this method to associate your evaluation with the corresponding span.
```

Comment:
```suggestion
`LLMObs.export_span()` can be used to extract the span context from a span. This method is helpful for associating your evaluation with the corresponding span.
```

Reply:
Also, is there a reason we moved this to after `submit_evaluation_for()`? To me it is more helpful to order these in the sequence that a user will be using them

Reply:
That reasoning is fine with me, thanks for clarifying!